it/v2/endpoints/DeleteAdditionalDirectorshipDividendsControllerISpec.scala

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright 2025 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v2.endpoints
+
+import com.github.tomakehurst.wiremock.stubbing.StubMapping
+import play.api.http.HeaderNames.ACCEPT
+import play.api.http.Status._
+import play.api.libs.json.Json
+import play.api.libs.ws.{WSRequest, WSResponse}
+import play.api.test.Helpers.AUTHORIZATION
+import shared.models.errors._
+import shared.services.{AuditStub, AuthStub, DownstreamStub, MtdIdLookupStub}
+import shared.support.IntegrationBaseSpec
+
+class DeleteAdditionalDirectorshipDividendsControllerISpec extends IntegrationBaseSpec {
+
+  "Calling the 'delete additional directorship dividends' endpoint" should {
+    "return a 204 status code" when {
+      "any valid request is made" in new HipTest  {
+
+        override def setupStubs(): StubMapping = {
+          AuditStub.audit()
+          AuthStub.authorised()
+          MtdIdLookupStub.ninoFound(nino)
+          DownstreamStub.onSuccess(DownstreamStub.DELETE, downstreamUri, NO_CONTENT)
```

Comment:
Use query params in the test here `DownstreamStub.onSuccess(DownstreamStub.GET, downstreamUri, Map("taxYear" -> downstreamTaxYear), OK, downstreamResponse)`
And on other error scenarios